vividus-plugin-rest-api/src/main/resources/org/vividus/http/attachment/api-message.ftl

```diff
@@ -87,10 +87,10 @@
                                 <#assign body = body?html>
                              </#if>
                              <div id="pretty" class="tab-pane fade  in active">
-                               <pre><code id = "pretty-code" class="${contentType}">${body}</code></pre>
+                               <pre><code id = "pretty-code" class="${contentType}"><#outputformat "XML">${body}</#outputformat></code></pre>
```

Comment:
it's not always `XML`, there could be a variety of formats: https://freemarker.apache.org/docs/dgui_misc_autoescaping.html#dgui_misc_autoescaping_outputformat